core/lib/zksync_core/src/api_server/web3/namespaces/zks.rs

```diff
@@ -646,4 +646,9 @@ impl ZksNamespace {
             storage_proof,
         })
     }
+
+    #[tracing::instrument(skip_all)]
+    pub async fn get_base_token_l1_address_impl(&self) -> Result<Address, Web3Error> {
```

Comment:
Nit: It doesn't make sense to make this function `async`; AFAIU, it's not like it's expected to become async in the future.

Reply:
Fixed!